### DISCUSSION TITLE
doctor  / patient speaker detection?

### DISCUSSION THREAD

**Discussion by scoutaroom** [deepgram/community#561] [General help] [answered]
Do you provide doctor-detection?
Something like that style, but it should recognize the speaker automatically ?
Patient: {text}
Doctor:{text}
Patient: {text}
Doctor:{text}
Patient: {text}
Doctor:{text}

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@scoutaroom We don't currently offer that type of functionality which we call `speaker identification`. We do have an open Feature Request for this, I'll link your post here to that one. 

https://github.com/orgs/deepgram/discussions/534
